pkg/status/status.go

```diff
@@ -247,15 +247,17 @@ func (s *status) updateStatus(previousStatus *configv1.ClusterOperatorStatus) er
 			}
 		}
 
-		_, err := s.configClient.ClusterOperators().UpdateStatus(s.clusterOperator)
-		if err != nil {
-			return fmt.Errorf("Error %v updating ClusterOperator", err)
-		}
 		// Log Conditions
-		log.Infof("[status] Current ClusterOperator conditions:")
+		log.Infof("[status] Attempting to set the ClusterOperator status conditions to:")
 		for _, statusCondition := range s.clusterOperator.Status.Conditions {
 			log.Infof("[status] ConditionType: %v\tConditionStatus: %v\tConditionMessage: %v", statusCondition.Type, statusCondition.Status, statusCondition.Message)
```

Comment:
While you are doing this, please remove the `\t`. They don't insert tabs in the installer output.

Reply:
Good to know!